src/components/generic/Banner/Banner.scss

```diff
@@ -0,0 +1,11 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-banner {
+  display: flex;
+  justify-content: center;
+  background: var(--wxc-banner-background);
+  color: var(--wxc-banner-text-color);
+  font-family: CiscoSans;
+  font-size: 16px;
+  height: 120px;
+  padding: 12px 0 84px 0;
```

Comment:
I don't think height and vertical paddings are both necessary, please leave only one of them

Reply:
please also use rem in this line

Reply:
rem:)